source/switch/main.cpp

```diff
@@ -512,6 +516,8 @@ int main(int argc, char *argv[]) {
 	uiAddSetting("Frameskip", &frameSkip, sizeof(frameSkipValues) / sizeof(frameSkipValues[0]), frameSkipNames);
 	uiAddSetting("Disable analog stick", &disableAnalogStick, 2, stringsNoYes);
 	uiAddSetting("Switch R and L buttons to ZR and ZL (switches speedhack button too)", &switchRLButtons, 2, stringsNoYes);
+	uiAddSetting("Use Nintendo Switch theme", &useSwitchTheme, 2, stringsNoYes);
+	uiAddSetting("Theme", &themeM, 2, themeOptions);
```

Comment:
I think it would be better to merge these two options into one where you can select System Default, Bright, Dark